src/main/io/serial.c

```diff
@@ -351,42 +351,42 @@ serialPort_t *openSerialPort(
 #endif
 #ifdef USE_UART1
         case SERIAL_PORT_USART1:
-            serialPort = uartOpen(USART1, rxCallback, baudRate, mode, options);
+            serialPort = uartOpen(UARTDEV_1, rxCallback, baudRate, mode, options);
```

Comment:
SERIAL_PORT_USARTx -> UARTDEV_x mapping may be implemented by macro `(identifier - SERIAL_PORT_USART1 + UARTDEV_1)`. Switch may be then simplified (using fallthrough to single uartOpen call)